mautic/mautic
# TPROD-290 The Kernel::getRootDir() and the kernel.root_dir parameter have been removed

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [n]
| New feature/enhancement? (use the a.x branch)      | [n]
| Deprecations?                          | [n]
| BC breaks? (use the c.x branch)        | [n]
| Automated tests included?              | [] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | https://mautic.atlassian.net/browse/TPROD-290


#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Upload an asset and verify id it's preview is fine.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11145"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
